cmd/config-seed/main.go

```diff
@@ -20,63 +20,70 @@ import (
 	"sync"
 
 	"github.com/edgexfoundry/edgex-go/internal"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
 	"github.com/edgexfoundry/edgex-go/internal/seed/config"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 )
 
 func main() {
+	var configProviderUrl string
 	var configDir, profileDir string
 	var dirCmd string
 	var dirProperties string
 	var overwriteConfig bool
 
+	flag.StringVar(&configProviderUrl, "configProvider", "", "Indicates to use Configuration Provider service at specified URL. Format: {type}.{protocol}://{host}:{port} ex: consul.http://localhost:8500.")
+	flag.StringVar(&configProviderUrl, "cp", "", "Indicates the service should use Configuration Provider.")
```

Comment:
Shouldn't the content of the help text for `cp` be the same as `configProvider`?

Reply:
Yes it should. Good catch.

Reply:
Fixed